views/includes/check-your-answers/presenter.html

```diff
@@ -1,19 +1,25 @@
-<h2 class="govuk-heading-l">Contact details</h2>
-<h3 class="govuk-heading-m">Who can we contact about this application?</h3>
-
 {% if pageParams.isRegistration %}
+  {% set contactDetailsSubText = "Who can we contact about this application?" %}
   {% set presenterNameChangeLink = OE_CONFIGS.PRESENTER_CHANGE_FULL_NAME %} 
   {% set presenterEmailChangeLink = OE_CONFIGS.PRESENTER_CHANGE_EMAIL %}
+  {% set nameFieldLabel = "Name" %}
+  {% set emailFieldLabel = "Email" %}
 {% else %} 
+  {% set contactDetailsSubText = "Who can we contact about this update application?" %}
   {% set presenterNameChangeLink = OE_CONFIGS.UPDATE_PRESENTER_CHANGE_FULL_NAME %} 
   {% set presenterEmailChangeLink = OE_CONFIGS.UPDATE_PRESENTER_CHANGE_EMAIL %}
+  {% set nameFieldLabel = "Full Name" %}
+  {% set emailFieldLabel = "Email address" %}
```

Comment:
I've a similar situation for 291. Registration content doesn't include "?" and update does. Did you run this past Matthew at all? I would wonder why a name field is different on registration vs update. Might be an oversight on the screenshots/prototype.

Reply:
Yes I did and I added it to the rational for this in our teams channel